src/WebJobs.Script.WebHost/Diagnostics/IEventGenerator.cs

```diff
@@ -14,7 +14,7 @@ public interface IEventGenerator
 
         void LogFunctionExecutionAggregateEvent(string siteName, string functionName, long executionTimeInMs, long functionStartedCount, long functionCompletedCount, long functionFailedCount);
 
-        void LogFunctionDetailsEvent(string siteName, string functionName, string inputBindings, string outputBindings, string scriptType, bool isDisabled);
+        void LogFunctionDetailsEvent(string siteName, string functionName, string inputBindings, string outputBindings, string scriptType);
```

Comment:
Same here - all the places where we were using IsDisabled before, I'd expect no changes to? We're just changing the implementation behind the scenes for that property.